### PR TITLE
percentage visitor wins and percentage home wins methods

### PR DESCRIPTION
**Pull request by andilovetto**
this has since been updated; we are no longer having the methods live within game_stats or game_factory. now the methods  are all living within stat_tracker and have been moved ~

instead of having the game_stats class hold the methods for game statistics, it seems better that the game_factory will hold the methods for the game statistics. the methods living in the game_stats class will just cause us to have to traverse farther down a filepath from stat_tracker and chain more things in order to call those methods.

### CODE REVIEW

**Review by ChrisAsercion — APPROVED** [review]
This looks good, moving forward I would say let's build methods in stat_tracker and then move them into their respective homes when refactoring later.

**Comment by andilovetto** [comment]
finished up writing and testing for the best offense method; moved all the methods that were written and living within game_factory into stat_tracker and changed the testing to reflect the same.

File: lib/game_factory.rb [hunk 1]
```diff
@@ -25,22 +22,4 @@ def create_games(path)
...
-    end
-    home_wins / @games.count * 100
-  end
-  
-  def percent_of_ties
-    ties = @games.count do |game|
-      game[:away_goals] == game[:home_goals]
-
-    end
-    (ties.to_f / @games.length).round(2)
```
Comment (JorjaF) [inline lib/game_factory.rb:44]: so this is where you deleted the percent of ties method on line 39-44

File: lib/stat_tracker.rb [hunk 2]
```diff
@@ -3,49 +3,81 @@
...
+    percentage_wins = (home_wins.to_f / @game_factory.games.count.to_f)
+    percentage_wins.round(2)
+  end
+  
+  def percent_of_ties
+    ties = @game_factory.games.count do |game|
+      game[:away_goals] == game[:home_goals]
+    end
+  end
+
```
Comment (JorjaF) [inline lib/stat_tracker.rb:53]: this is were you added that percent_of_ties method back in but some how you did not in the full method.

**Review by JorjaF — CHANGES_REQUESTED** [review]
I found where in your deleted files where you took out my method and then in your add files you entered it back in but it was missing 3 lines. can you just go ahead and delete this method completely because I will just move it to stat_tracker

File: lib/stat_tracker.rb (hunk 2 above)
Comment (ChrisAsercion) [reply lib/stat_tracker.rb:53]: Is it just the one line  ` (ties.to_f / @games.length).round(2)` that this method needs? I don't see why it needs to be deleted out right and re-added with another branch. We can go in and add that line and then commit and this branch should work if I am understanding the problem correctly.

**Review by JorjaF — APPROVED** [review]

**Pull request merged**